python/lsst/meas/algorithms/computeExPsf.py

```diff
@@ -0,0 +1,305 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.pex.config import ChoiceField, Config, Field, FieldValidationError
+from lsst.pipe.base import Task
+import lsst.pipe.base as pipeBase
+import treecorr
+import copy
+import numpy.typing as npt
+
+
+__all__ = ("ComputeExPsfTask", "TreecorrConfig")
+
+
+class TreecorrConfig(Config):
+    """A Config class that holds some of the parameters supported by treecorr.
+
+    The fields in this class correspond to the parameters that can be passed to
+    any calls to `treecorr` methods, including catalog creation and two-point
+    correlation function calculations. The default values set for the fields
+    are identical to the default values set in v4.2 of `treecorr`.
```

Comment:
It'd be good to check if the defaults are consistent with the  version of `treecorr` available in the `rubin-env` / shared stack at USDF.

Reply:
I did and I made the corrections if needed.